frontend/src/js/snack-message/actions.ts

```diff
@@ -1,11 +1,14 @@
 import { ActionType, createAction } from "typesafe-actions";
 
+import { SnackMessageTypeT } from "./reducer";
+
 export type SnackMessageActions = ActionType<
   typeof setMessage | typeof resetMessage
 >;
 
 export const setMessage = createAction("snack-message/SET")<{
   message: string | null;
+  notificationType: SnackMessageTypeT;
```

Comment:
how about just calling it `type`? 
```suggestion
  type: SnackMessageType;
```